hypertrace-trace-enricher/hypertrace-trace-enricher-impl/src/test/java/org/hypertrace/traceenricher/enrichment/enrichers/BackendEntityEnricherTest.java

```diff
@@ -82,6 +82,22 @@ public void test_EnrichTrace_noBackendResolved() {
     Assertions.assertNull(EnrichedSpanUtils.getBackendId(e));
   }
 
+  @Test
+  public void test_EnrichEvent_backendOperationEvent() {
+    Event e = createApiEntryEvent(EVENT_ID).build();
+    StructuredTrace trace = createStructuredTrace(TENANT_ID, e);
+    enricher.enrichEvent(trace,e);
+    Assertions.assertNull(EnrichedSpanUtils.getBackendOperation(e));
+  }
+
+  @Test
+  public void test_EnrichEvent_validBackendOperationEnrichment() {
+    Event e = createApiEntryEvent(EVENT_ID).setEnrichedAttributes(createNewAvroAttributes(Map.of(BACKEND_OPERATION_ATTR, "select"))).build();
```

Comment:
The test is not doing what it's supposed to do. The test should ideally be adding attributes like `db.operation` in the even attributes, then call `enricher.enrichEvent(trace,e);` and then assert the enriched attribute